pymc4/flow/executor.py

```diff
@@ -104,6 +104,7 @@ class SamplingState:
         "distributions",
         "potentials",
         "deterministics",
+        "deterministic_distributions",
```

Comment:
Could there be a better naming for this attribute though?

Reply:
What is a deterministic distribution?

Reply:
I totally agree the name is not well chosen. How about `deterministic_variables`, `deterministic_objects` or  `deterministic_instances` ?

Reply:
More specifically, what's the difference between `determinstics` and `deterministic_distributions`?

Reply:
`deterministics` is the tensor i.e. `pm.Deterministic.get_value()` and `deterministic_distributions` is the class instance `pm.Deterministic`

See [L644-L645 ](https://github.com/pymc-devs/pymc4/blob/30b39e411b6a44c6abc9b271c3ee43b7536be342/pymc4/flow/executor.py#L644-L645)

Reply:
In that case, I would change `deterministics` to `deterministics_values`  and `deterministic_distributions` to `deterministics`. That would seem more consistent.

Reply:
> In that case, I would change `deterministics` to `deterministics_values` and `deterministic_distributions` to `deterministics`. That would seem more consistent.

Done, I replaced more or less every occurrence of `deterministics` with `deterministics_values`. Was kinda all over the place but should work as far as I can see.